resource-aggregate/pb/commands.proto

```diff
@@ -6,19 +6,14 @@ import "github.com/plgd-dev/cloud/resource-aggregate/pb/resources.proto";
 
 option go_package = "github.com/plgd-dev/cloud/resource-aggregate/commands;commands";
 
-message AuthorizationContext {
-    string device_id = 1;
-}
-
 message CommandMetadata {
     string connection_id = 1;
     uint64 sequence = 2;
 }
 
 message AuditContext {
     string user_id = 1;
-    string device_id = 2;
-    string correlation_id = 3;
+    string correlation_id = 2;
```

Comment:
Why did you drop device_id from the audit context? then you need to search in every event and command for a device id based on contract. That's not good.

Reply:
device_Id is the part of resource_id. so this is not true